### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by ckarli**
Django 5.0 Support

[//]: # (Thanks for helping us out! Cache invalidation sucks, so your help is greatly appreciated!)

## Description

[//]: # (What're you proposing?)

## Rationale

[//]: # (Why should we implement it?)

### CODE REVIEW

**Comment by ckarli** [comment]
Consolidated the pull requests.

**Comment by Andrew-Chen-Wang** [comment]
please also deprecate py 3.7 and include py 3.12. much appreciated

**Comment by jgillard** [comment]
I see from the last commit that updated Django support, that at least these will need updating also: README.rst, CHANGELOG.rst, docs/quickstart.rst, setup.py, tox.ini. I've never worked on this project, so am not sure what other changes might be required.

https://github.com/noripyt/django-cachalot/commit/269798692ed73ffad41d3aec100d7ec0e796969f

**Comment by ckarli** [comment]
Updated the files @jgilliard mentioned.

File: tox.ini [hunk 1]
```diff
@@ -13,10 +15,12 @@ basepython =
     py39: python3.9
     py310: python3.10
     py311: python3.11
+    py312: python3.12
 deps =
     django3.2: Django>=3.2,<4.0
     django4.1: Django>=4.1,<4.2
     django4.2: Django>=4.2,<4.3
+    django5.0: Django>=5.9,<5.1
```
Comment (Andrew-Chen-Wang) [inline tox.ini:23]:
```suggestion
    django5.0: Django>=5.0,<5.1
```

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]

**Comment by ckarli** [comment]
Whops. Typo fixed.

**Review by Andrew-Chen-Wang — APPROVED** [review]

**Comment by Andrew-Chen-Wang** [comment]
thanks!

**Comment by Andrew-Chen-Wang** [comment]
Hm upon second inspection, the CI itself is not actually running Django 5.0. It's running 4.2.9?

**Pull request merged**